demos/JWST/S5_fit_par_template.epf

```diff
@@ -70,6 +70,25 @@ u2          0.84            'free'          0           1           U
 # osc_t0          0.3163  'fixed'         0            1              U
 # osc_t1          0.3163  'free'          0.3          0.4            U
 
+# ------------------
+# ** Star Spot parameters **
+# each spot needs rad, lat, lon. see docs for details!
+# for fleck: can only have one spot contrast (spotcon0), spotnpts is the number of temporal points to evaluate at, spotrad is relative to star
+# for starry: assign one spotcon per spot (spotcon0, spotcon1, etc), spotres is the ydeg of the star map, spotrad is in degrees
+# NOTE: fleck contrast = 1 - starry contrast
+# ------------------
+# spotstari    90       'fixed'         90        1       N
```

Comment:
While this parameter is set to fixed in the template, the demonstrated prior (should users choose to set the parameter to free) is 90 +/- 1 degree which seems excessively constraining. Wouldn't a prior of U(0,90) or something be more fitting if someone chose to fit for this parameter?

Reply:
This parameter only matters if the star is rotating (fleck's "slow" mode) otherwise the inclination is irrelevant because the lat/lon of spot can change to accommodate it. I don't know that people would fit for this since information from R-M measurements would be more informative than spots in transits.